velox/expression/VectorFunction.cpp

```diff
@@ -37,6 +38,22 @@ std::optional<std::vector<FunctionSignaturePtr>> getVectorFunctionSignatures(
       });
 }
 
+std::shared_ptr<const Type> resolveVectorFunction(
```

Comment:
Moved this to experession/VectorFunction from functions/FunctionRegistry since velox/functions depends on expressions and this creates a circular dependency.